tests_e2e/pipeline/scripts/execute_tests.sh

```diff
@@ -86,18 +86,27 @@ sudo find "$LOGS_DIRECTORY" -exec chown "$USER" {} \;
 #
 # Move the relevant logs to the staging directory
 #
+# Move the logs for failed tests to a temporary location
```

Comment:
Why do we move failed tests to a temporary location here?

Reply:
we move them back on line 105

the temp location makes it easy to separate them from other logs